src/client/sandbox/node/element.js

```diff
@@ -161,9 +162,28 @@ export default class ElementSandbox extends SandboxBase {
         var overridedGetAttributeCore    = (el, attr, ns) => this._overridedGetAttributeCore(el, attr, ns);
         var overridedSetAttributeCore    = (el, attr, value, ns) => this._overridedSetAttributeCore(el, attr, value, ns);
         var overridedRemoveAttributeCore = (el, ns, arg) => this._overridedRemoveAttributeCore(el, ns, arg);
+        var overrideTextNode             = (node, parentNode) => {
```

Comment:
> I'd prefer to move all these arrows to the instance methods BTW, I don't see any reason to have this shortcuts here.

Code structure:
```javascript
_createOverridedMethods () {
  //shortcuts
         /*dependent sandbox instance method*/
        var overrideNewElement           = el => this.nodeSandbox.overrideDomMethods(el);
        /* instance method */
        var onElementAdded               = el => this._onElementAdded(el);
        /* instance method */
        var onElementRemoved             = el => this.onElementRemoved(el);
        var removeFileInputInfo          = el => ElementSandbox._removeFileInputInfo(el);
        /* instance method */
        var overridedGetAttributeCore    = (el, attr, ns) => this._overridedGetAttributeCore(el, attr, ns);
        var overridedSetAttributeCore    = (el, attr, value, ns) => this._overridedSetAttributeCore(el, attr, value, ns);
        var overridedRemoveAttributeCore = (el, ns, arg) => this._overridedRemoveAttributeCore(el, ns, arg);
        var insertBeforeShadowUIRoot     = el => this.shadowUI.insertBeforeRoot(el);
        var processTextNodeContent       = (node, parentNode) => this._processTextNodeContent(node, parentNode);
  this.overridedMethods = {
     insertCell: function () {
            overrideNewElement(this);
     },
     ...
  };
}

```

`this` inside overrided function is a html element.
We need to call sandbox instance methods and dependent sandboxes instance methods inside overrided functions in many places.